# BlastAutoPython

## Introduction

For gaining more insight into found spacers by Pilercr program, we aim to automate the Blast procedure and then blast all of the found spacers against NCBI nucleotide database.
First we start by extracting all spacers of a genome from Pilercr output.

## 0 Cleaning up the pilercr output and ectract all spacers in a genome and make a multi-fasta file with spacers in each line.

Example: 

>AC_000091_0
CCGGGGGATAATGTTTACGGTCATGCGCCCCC
>AC_000091_1
TGGGCGGCTTGCCTTGCAGCCAGCTCCAGCAG
>AC_000091_2
AAGCTGGCTGGCAATCTCTTTCGGGGTGAGTC
>AC_000091_3
TAGTTTCCGTATCTCCGGATTTATAAAGCTGA
>AC_000091_4
GCAGGCGGCGACGCGCAGGGTATGCGCGATTCG
>AC_000091_5
GCGACCGCTCAGAAATTCCAGACCCGATCCAAA
>AC_000091_6
TCAACATTATCAATTACAACCGACAGGGAGCC
>AC_000091_7
AGCGTGTTCGGCATCACCTTTGGCTTCGGCTG
>AC_000091_8
TGCGTGAGCGTATCGCCGCGCGTCTGCGAAAG
>AC_000091_9
TCTAAAAGTAT
>AC_000091_10
CTGTTTTCGCAAATCTATGGACTATTGCTATTC
>AC_000091_11
GGGCGCACGGAATACAAAGCCGTGTATCTGCTC
>AC_000091_12
TGGCTCTGCAACAGCAGCACCCATGACCACGTC
>AC_000091_13
GAAATGCTGGTGAGCGTTAATGCCGCAAACACA
>AC_000091_14
ATTACGCCTTTTTGCGATTGCCCGGTTTTTGCC
>AC_000091_15
TCTAAACATA


## 1 Automating the Blast procedure using BioPython package

(... In progress ...)